benchmark_utils/deconv_utils.py

```diff
@@ -22,15 +22,15 @@ def perform_nnls(signature: pd.DataFrame,
     Paramaters
     ----------
     signature: pd.DataFrame
-        Signature matrix of shape (n_features, n_components)
+        Signature matrix of shape (n_genes, n_cell_types)
     adata_pseudobulk: ad.AnnData
-        AnnData object of shape (n_samples, n°)
+        AnnData object of shape (n_samples, n_genes) | relative counts
```

Comment:
You say relative counts here but on line 33 you change from relative counts to counts ?